README.md

```diff
@@ -41,7 +41,8 @@ Optional prerequisites can be skipped for reduced functionality.
 |                        | `OBSERVATION_PORTAL_API_TOKEN`| The API Token for an admin of the observation-portal                                   | _`Empty string`_                                 |
 |                        | `REDIS_URL`             | The url of the redis cache (or the linked container name)           | `redis://redis`                                                 |
 | Kernel Settings       | `KERNEL_ALGORITHM`     | Algorithm code for ORTools to use. Options are `CBC`, `SCIP`, `GLPK`, and `GUROBI`      | `SCIP`                                                 |
-| Kernel Settings       | `KERNEL_FALLBACK_ALGORITHM`     | Fallback algorithm in case main choice fails or throws an exception. Options are `CBC`, `SCIP`, `GLPK`, and `GUROBI`      | `SCIP`                                                 |
+|                       | `KERNEL_FALLBACK_ALGORITHM`     | Fallback algorithm in case main choice fails or throws an exception. Options are `CBC`, `SCIP`, `GLPK`, and `GUROBI`      | `SCIP`                                                 |
+|                       | `KERNEL_PARAMS`     | Set Kernel specific params within ORTools. Only modify this if you know what you are doing as these values are heavily dependent on the underlying algorithm. An example of this would be `Threads 2\nMethod 3` for the GUROBI Kernel to set the number of threads it uses to 2 and the method to concurrent.     | _`Empty string`_                                                 |
```

Comment:
Great idea! Can you add a little blurb that this env is passed as is to the `SetSolverSpecificParametersAsString` function, if set. People can then look that up in the ortools docs.